app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -21,7 +21,8 @@ def build_request_body
             account_last_name: applicant[:last_name],
             account_telephone: '', # applicant[:phone], decision was made not to send phone
             account_drivers_license_number: applicant[:state_id_number].gsub(/\W/, ''),
-            account_drivers_license_issuer: applicant[:state_id_jurisdiction].to_s.strip,
+            account_drivers_license_type: "us_dl",
```

Comment:
i think rubocop is gonna yell about this one
```suggestion
            account_drivers_license_type: 'us_dl',
```